samza-core/src/test/java/org/apache/samza/table/remote/TestRemoteTableDescriptor.java

```diff
@@ -176,8 +176,10 @@ private void doTestDeserializeReadFunctionAndLimiter(boolean rateOnly, boolean r
       Assert.assertTrue(!rlPuts || rwTable.writeRateLimiter != null);
     }
 
-    ThreadPoolExecutor callbackExecutor = (ThreadPoolExecutor) rwTable.asyncHelper.callbackExecutor;
-    Assert.assertEquals(10, callbackExecutor.getCorePoolSize());
+    if (numRateLimitOps > 0) {
```

Comment:
this is not needed, callback executor is not tied to rate limiting. I'll remove this check in the next update.